proto/commonfate/control/filters/v1alpha1/filters.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package commonfate.control.filters.v1alpha1;
+
+import "commonfate/authz/v1alpha1/evaluation.proto";
+import "commonfate/authz/v1alpha1/read_only.proto";
+import "commonfate/entity/v1alpha1/eid.proto";
+import "google/protobuf/timestamp.proto";
+
+// Filters events with a particular authorization decision
+message DecisionFilter {
+  commonfate.authz.v1alpha1.Decision decision = 1;
+}
+
+// Filters event by a particular key/value tag associated when them.
```

Comment:
Could you please update all the comments to be generic, now that these filters don't directly relate to filtering log events